internal/common/baseclient/baseclient.go

```diff
@@ -93,6 +101,10 @@ func (c *BaseClient) GetID() shared.ClientID {
 	return c.id
 }
 
+func (c BaseClient) Initialise(ServerReadHandle) {}
```

Comment:
we should impleemnt this for the baseclient?

and add a variable for the BaseClient struct for it.

Use pointer rcvers btw

Reply:
Is there any reason to keep that in the base client? I could add it but it won't be used. 
I thought baseclient is supposed to have "null" implementations.

Reply:
nope, baseclient is supposed to implement a working client as a basis for extensions

Reply:
ptr receivers done

Reply:
> nope, baseclient is supposed to implement a working client as a basis for extensions

Makes sense. Keeping the ServerReadHandle in the baseclient.